cvmfs/bundle_mgr.h

```diff
@@ -0,0 +1,34 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#ifndef CVMFS_BUNDLE_MGR_H_
+#define CVMFS_BUNDLE_MGR_H_
+#include "duplex_fuse.h"  // fuse_ino_t
+#include "mountpoint.h"   //MountPoint*, FileSystem*
+
+namespace cvmfs {
+/*
+ * functions defined inside /cvmfs/cvmfs.cc
+ */
+bool GetPathForInode(const fuse_ino_t ino, PathString *path);
+bool GetDirentForInode(const fuse_ino_t ino, catalog::DirectoryEntry *dirent);
```

Comment:
I think we should avoid exposing these functions (which are quite closely coupled to the fuse logic).